### ISSUE TITLE
daohaus stock distribution

### ISSUE THREAD

**Issue by cheo0202** [ShenaniganDApp/SHEiP#12]
<REMOVE>---
SHEiP: <>
title: <daohaus stock distribution>
author: <jose coronado (@cheo0202>
discussions-to: <N/A>
status: <Idea>
category: <Informational>
type: < Community >
created: <2021-10-29>
edited: <N/A>
requires: <N/A>
replaces: <N/A>
<REMOVE>---

## Simple Summary
<information on Daohaus shares>
information on Daohaus shares

## Abstract
<It is proposed to create a system of distribution of shares of the daohaus so that the community has the right to vote in the course of the project.>
It is proposed to create a system of distribution of shares of the daohaus so that the community has the right to vote in the course of the project.

## Motivation
<The community has always wanted to be involved in having the right to make decisions within SHE, it is for this reason that it occurred to me that they not only participate in the snapshots but also have a participation in the core of the project>
Motivation goes here.

## Specification
<My proposal is based on the sale of shares within daohaus which the funds raised would go to the contribution of liquidity or what the project is needing>
My proposal is based on the sale of shares within daohaus which the funds raised would go to the contribution of liquidity or what the project is needing

## Rationale

The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is works in other environments. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion

## Backwards Compatibility

Backwards compatibility goes here.

## Implementation
<Rewards can start calculating from the day this proposal passes>
Rewards can start calculating from the day this proposal passes

## Security Considerations

Security considerations go here.

## Copyright
Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).

**Comment by youngkidwarrior** [comment]
First off @cheo0202, my apologies for my delayed response on this. It's been getting pushed back on my list again and again.

If I understand your proposal correctly, you'd like PRTCLE holders to receive voting shares in the DAOHaus DAO. As it stands now, the DAOHaus DAO acts as a multisig of 9 major contributors of the project.

Here is a screenshot of the current voting share distribution: 
![image](https://user-images.githubusercontent.com/36084652/140670860-3fea1ead-fd49-49fe-8c88-5d39bd5dedb9.png)


Let me explain the reasoning for doing this. Back when the DAO was on Aragon, we had the problem of reaching its 50% quorum more than once. DAOHaus allowed us to outsource token voting to http://snapshot.org and reduce the reliance on quorum for placing proposals into actuation. This move was a major decision and included a bunch of trade offs.

Pros: 
Faster, better maintained interface
Gasless voting
LP Voting

Cons:
DAO is now a public multisig
Reduced decentralization

So that means I am open to giving other members of the community a voting share in exchange for PRTCLE or XDAI. However, it is required of that person to be active in voting in the DAO when called upon. There is no monetary benefit to holding voting shares. Anyone who makes and voted on proposals is charged PRTCLE and time for their efforts. So, those that receive voting shares must give their time, money, and energy with the understanding that there will be no payment in return.


I think a better solution would be to use LOOT shares to represent those who are interesting in owning part of the DAO, without voting. LOOT shares give ownership over the treasury, without requiring voting participation. With Moloch v3, PRTCLE Holders can stake their PRTCLE in exchange for LOOT shares.

**Comment by cheo0202** [comment]
I think it's very good about the LOOP actions. There would be no possibility of being bought with the HAUS tokans so the funds would go to a pool, which would strengthen the PRTCLE.

**Comment by cheo0202** [comment]
Another thing that occurs to me is to spread the information about these terms and if any of the community earns the merit of having a share with the right to vote that is granted

**Comment by diegoleal8802** [comment]
It would be nice to have some snapshots. on this topic if you want actions loop
